packages/devtools_app/assets/scripts/inspector_polyfill_script.dart

```diff
@@ -94,43 +105,45 @@ String addServiceExtensions() {
         return entry;
       }
     }
-    return null;
+    throw 'Enum value $name not found';
   }
 
   Future<Map<String, dynamic>> getLayoutExplorerNode(
       Map<String, String> parameters) {
-    final String id = parameters['id'];
-    final int subtreeDepth = int.parse(parameters['subtreeDepth']);
-    final String groupName = parameters['groupName'];
-    Map<String, Object> result = {};
+    final id = parameters['id'];
+    final subtreeDepth = int.parse(parameters['subtreeDepth'] as String);
+    final groupName = parameters['groupName'];
+    var result = <String, dynamic>{};
```

Comment:
why the change from Object to dynamic?

Reply:
this has to be dynamic because that is now the type on the Flutter side. Before null safety, Object and dynamic basically meant the same thing so you could use them interchangeably in generics without getting warnings. With null safety they are different because
`Object?` and `Object` are different.